foodhub/foodhub/settings.py

```diff
@@ -0,0 +1,166 @@
+"""
+Django settings for foodhub project.
+
+Generated by 'django-admin startproject' using Django 4.1.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+import datetime
+from pathlib import Path
+from datetime import timedelta
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-j#gy^4ppz59i&w029%gf#*6nn4e0&ovbt6j%ic6kjj&z2yx#=u'
```

Comment:
Try to always have secrets stored & loaded from ENV

Reply:
Sure i will create new .env file and separate Secret Key, Database Configuration and rest of the things